README.md

```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
 
 You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
 
-### Connect with the Pieces Community
+### Connect/Disconnect your cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect your cloud` or `Pieces: Disconnect your cloud` command in your command palette.(Login required!)
```

Comment:
Can these commands be "Pieces: Disconnect Personal Cloud" and "Pieces: Connect Personal Cloud" ?